js/miniautures.js

```diff
@@ -0,0 +1,28 @@
+import  {urlForExport, likesForExport, commentsForExport} from './data.js';
+
+const pictures = document.querySelector ('.pictures');
+const pictureTemplate = document.querySelector ('#picture')
+  .content
+  .querySelector('a');
+
+const drawPicture = function () {
+  const newPicture = pictureTemplate.cloneNode(true);
+
+  const pictureSrcElement = newPicture.querySelector('src');
+  pictureSrcElement.textContent = urlForExport;
```

Comment:
у объекта ноды изображение свойство src в него записываем урл нашей картинки, которую получили в качестве аргумента функции
```suggestion
 pictureSrcElement.src = picture.url;
```